native/effektio/Cargo.toml

```diff
@@ -68,7 +68,7 @@ android_logger = "0.11"
 tokio = { version = "1", features = ["rt-multi-thread", "parking_lot"] }
 tracing-android = "0.2"
 
-[target.'cfg(all(not(target_os = "android"),not(target_arch = "wasm32"), not(target_os = "ios")))'.dependencies]
+[target.'cfg(all(not(target_os = "android"), not(target_arch = "wasm32"), not(target_os = "ios")))'.dependencies]
```

Comment:
what is this for?

Reply:
`not(target_os = "ios")` is prefixed by space, but `not(target_arch = "wasm32")` is not prefixed by space.
I just wanted to correct this file for better readability.